app/controllers/merchant/items_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Merchant::ItemsController < Merchant::BaseController
   def index
-    @items = current_user.items
+    @item = Item.merchant_items(current_user)
   end
```

Comment:
For this one... is @item a group of items? Shouldn't we stick with the plural?

Reply:
You're right, I think I just missed that when I rewrote it. I'll make the change right now.

Reply:
Fixed it. Thanks, Tim!

Reply:
Thanks Chris!

Reply:
I'm a little curious why we changes from the current_user.items to this new method. Sorry if i'm not getting something but the original seems more straightforward.

Reply:
I'm not sure if the current_user.items method was here when I wrote this but you're right Peter, it is easier to just access it through the current user's items. I think I was stuck in ActiveRecord mode when I wrote this and forgot that there was an easier way. I'll switch it back and remove the unnecessary model test for this method.